catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -383,4 +383,11 @@
         <feature>catalog-core</feature>
     </feature>
 
+    <feature name="catalog-core-ftp" install="manual" version="${project.version}"
+             description="FTP endpoint for ingesting files into the DDF catalog. Supports PUT and MPUT operations only. Avoids the extra IO overhead of otherwise having to temporarily write the file to the file system, for example, like the DirectoryMonitor.">
+        <feature prerequisite="true">catalog-app</feature>
+        <bundle>mvn:ddf.catalog.core/catalog-core-ftp/${project.version}</bundle>
+        <bundle>mvn:org.apache.mina/mina-core/2.0.13</bundle>
```

Comment:
You should create a property for the mina version since you use it in a few places and it will make it much easier to update.